src/style/design-tokens/carbon-scoped-token-provider/carbon-scoped-tokens-provider.component.spec.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { mount } from "enzyme";
+import { aegeanTheme, mintTheme, noTheme, sageTheme } from "../../themes";
+import CarbonScopedTokensProvider from "./carbon-scoped-tokens-provider.component";
+import { noThemeSnapshot } from "../../../__spec_helper__/enzyme-snapshot-helper";
+
+const render = (theme) => mount(<CarbonScopedTokensProvider theme={theme} />);
+
+describe("CarbonScopedTokensProvider", () => {
+  it("should render css variables for none theme", () => {
+    const provider = render(noTheme);
```

Comment:
why we can't make one `it` test using `.each` and just create an array with `themes` and just use it in one `it`?

Reply:
Well, we can :). I just haven't thought about this.